account_invoice_reminder/models/invoice.py

```diff
@@ -0,0 +1,16 @@
+from odoo import fields, models
+
+
+class account_invoice(models.Model):
+    _inherit = "account.invoice"
+
+    reminder = fields.Selection(
+        [
+            ("1", "Premier rappel"),
+            ("2", "Second rappel"),
+            ("3", "Troisieme rappel"),
+            ("4", "Recommande"),
+        ],
+        string="Rappel",
+    )
+    last_reminder_date = fields.Date(string="Date du dernier rappel")
```

Comment:
shouldn’t these be in english, with a french translation?

Reply:
@huguesdk I think you're right. How does 'recommande' translate?